glusterfs.spec.in

```diff
@@ -260,12 +266,14 @@ BuildRequires:    libasan
 %if 0%{?_with_tsan:1} && !( 0%{?rhel} && 0%{?rhel} < 7 )
 BuildRequires:    libtsan
 %endif
+
 BuildRequires:    bison flex
 BuildRequires:    gcc make libtool
 BuildRequires:    ncurses-devel readline-devel
 BuildRequires:    libxml2-devel openssl-devel
 BuildRequires:    libaio-devel libacl-devel
 BuildRequires:    python%{_pythonver}-devel
+BuildRequires:    gperftools-devel
```

Comment:
@msaju - that will require changes in Gluster-builds Dockerfiles as well.

Reply:
@mykaul  - noted, will do the needful